core/storage/feedback/gae_models.py

```diff
@@ -88,18 +97,42 @@ def generate_new_thread_id(cls, exploration_id):
 
     @classmethod
     def generate_full_thread_id(cls, exploration_id, thread_id):
+        """Generates a full thread ID which is unique globally.
+
+        Args:
+            exploration_id: str. The exploration id the thread belongs to.
+            thread_id: str. Thread ID of given thread.
```

Comment:
There is no "given thread" -- "given" implies that the thread is one of the args. A better docstring would probably be "The thread ID."